### PR TITLE
2_6: Answer checker

### PR DESCRIPTION
**Pull request by Tenzijn**
1. style.css made changes  with class name  answerList and answerList_item
2. Added answer checker

### CODE REVIEW

File: src/pages/questionPage.js [hunk 1]
```diff
@@ -39,12 +39,11 @@ export const initQuestionPage = () => {
   userInterface.appendChild(questionNumberTrackerComponent);
 
   /**************************Local storage ************************************* */
-  setLocalStorage(currentQuestionNumber);
+  setLocalStorage(currentQuestionNumber, currentQuestion.correct);
```
Comment (NouranMahmoud) [inline src/pages/questionPage.js]: Remove this it will be covered in 1_4

File: src/utils/checkAnswer.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { quizData } from '../data.js';
+
+export const checkAnswer = (questionNum) => {
+  const answer = localStorage.getItem(questionNum);
```
Comment (NouranMahmoud) [inline src/utils/checkAnswer.js:4]: Replaces `questionNum` with `questionId`

File: src/utils/checkAnswer.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  const correctAnswer = quizData.questions[questionNum - 1].correct;
```
Comment (NouranMahmoud) [inline src/utils/checkAnswer.js:5]: `questionNum - 1` to be questionId

File: src/utils/setLocalStorage.js [hunk 4]
```diff
@@ -1,8 +1,29 @@
...
+        document.getElementById(answer.id).classList.add('correct');
+      } else {
+        console.log('wrong');
+        document.querySelectorAll('li').forEach((answer) => {
+          answer.classList.remove('correct');
+          answer.classList.remove('wrong');
+        });
+        document.getElementById(answer.id).classList.add('wrong');
+        document.getElementById(correctAnswer).classList.add('correct');
+      }
```
Comment (NouranMahmoud) [inline src/utils/setLocalStorage.js]: @ErhanKRL You can use this code to style wrong and correct answers

File: src/utils/checkAnswer.js (hunk 2 above)
Comment (Tenzijn) [reply src/utils/checkAnswer.js:4]: questionNum is question id that is passed form other component, So i think that is more appropriate.

File: src/utils/checkAnswer.js (hunk 3 above)
Comment (Tenzijn) [reply src/utils/checkAnswer.js:5]: if we use question id on this then we have loop through all the question to find the match and then get the correct answer from matched id. I think this is more fast and convenient way.

**Pull request closed without merging**